eth/ethconfig/config.go

```diff
@@ -60,7 +60,7 @@ var LightClientGPO = gasprice.Config{
 // Defaults contains default settings for use on the Ethereum main net.
```

Comment:
Ethereum mainnet has `1`, so either the change below is moot, or this piece of text is wrong